pkg/frontend/mysql_cmd_executor.go

```diff
@@ -1386,6 +1386,19 @@ func (mce *MysqlCmdExecutor) doComQuery(sql string) (retErr error) {
 			if err != nil {
 				goto handleFailed
 			}
+		case *tree.Insert:
+			_, ok := st.Rows.Select.(*tree.ValuesClause)
+			if ok && usePlan2 {
+				selfHandle = true
+				affectedRows, err := mce.handleInsertValues(st, epoch)
+				if err != nil {
+					goto handleFailed
+				}
+				err = proto.sendOKPacket(affectedRows, 0, 0, 0, "")
```

Comment:
put the response into the `handleInsertValues` just like `handleSetVar`. If `handleInsertValues` succeeds , it have sent the OK response after returing. If`handleInsertValues` fails, it just return err